ui/e2e/instances/details/index.spec.ts

```diff
@@ -250,3 +259,194 @@ test("the diagram on the instance page changes appearance dynamically", async ({
+
+  await resizeButton.click();
+
+  const maximizedWidth = (await inputOutputPanel.boundingBox())?.width;
+  if (minimizedWidth === undefined || maximizedWidth === undefined) {
+    throw new Error("could not get width of diagram panel");
+  }
+  expect(
+    maximizedWidth / minimizedWidth,
+    "The panel is significantly bigger after maximizing"
+  ).toBeGreaterThan(1.5);
+
+  await resizeButton.hover();
+  await expect(
+    page.getByText("minimize output"),
+    "it shows the correct text when hovering over the resize button"
+  ).toBeVisible();
+
+  await page.reload();
+
+  const currentWidthAfterReload = (await inputOutputPanel.boundingBox())?.width;
+  expect(
+    currentWidthAfterReload,
+    "after reloading the page, the panel is still maximized"
+  ).toEqual(maximizedWidth);
+
+  await resizeButton.click();
+  await inputButton.click();
+
+  await expect(textarea, "the text shows the expected input").toHaveText(
+    expectedInput
+  );
+
+  await outputButton.click();
+
+  await expect(textarea, "the text shows the expected output").toHaveText(
+    expectedOutput
+  );
+
+  // for writing to the clipboard and for reading from the clipboard we need to grant permissions here - if we are not allowing it universally in the playwright.config.ts
+  page.context().grantPermissions(["clipboard-write", "clipboard-read"]);
+
+  await copyButton.click();
+
+  expect(await page.evaluate(() => navigator.clipboard.readText())).toEqual(
+    expectedOutputCopy
+  );
+
+  await inputButton.click();
+  await copyButton.click();
+
+  expect(await page.evaluate(() => navigator.clipboard.readText())).toEqual(
+    expectedInput
+  );
+});
+
+test("the output shows event notifications", async ({ page }) => {
+  const newInstance = createInstance({
+    namespace,
+    path: failingWorkflowName,
+  });
+
+  await expect(newInstance, "wait until process was completed").toBeDefined();
+  const instanceId = (await newInstance).instance;
+  await page.goto(`/${namespace}/instances/${instanceId}`);
+
+  const inputOutputPanel = page.getByTestId("inputOutputPanel");
+
+  await expect(inputOutputPanel).toBeVisible();
+
+  const outputButton = inputOutputPanel
+    .getByRole("tablist")
+    .locator("button")
+    .nth(1);
+
+  const header = page.getByTestId("instance-header-container");
+  const textarea = inputOutputPanel.locator(".view-lines");
+
+  const runningInstanceOutput = "The workflow is still running";
+  const failedInstanceOutput = '{    "result": "an error occurred"}';
+
+  await outputButton.click();
+
+  await expect(
+    inputOutputPanel,
+    "the output shows the text for the status running instance"
+  ).toContainText(runningInstanceOutput);
+
+  await expect(
+    header.locator("div").first(),
+    "the badge failed is visible"
+  ).toContainText("failed");
+
+  await expect(
+    textarea,
+    "the output shows the text for the status failed instance"
+  ).toHaveText(failedInstanceOutput);
+});
```

Comment:
I have some suggestions according to the test description and the name of the tests inside. I think that could be a bit more expressive. 

These are just suggestions. Feel free to adjust them if oyu like:

Test name: "The output is shown when the workflow finished running"? 

First state: "The output shows a note, that the workflow did not generate an output yet"

Last state "When the workflow finished, the generated output is shown in the panel"

I also wonder if we should use a workflow that succeeds instead of a failed on. I read this test a couple of times because I thought this was somehow related to the workflow being failed. But this is not mandatory for me, I leave it up to you if you want to change it.